### PR TITLE
[新增]: 建立新的 module ：bear、footprint、reporter (draft PR)

### PR DESCRIPTION
**Pull request by yanghaochang104**
# 新增
- 依據 #44 ，新增黑熊、足跡、回報者的table module
- 此為 draft PR，由於我的安裝環境有問題，還有卡在磁碟空間不夠ＸＤ，所以沒辦法用python manage.py migrate  去generate migration files，尚待排除，我找個時間清一下

### CODE REVIEW

**Review by Neilxx — APPROVED** [review]

**Comment by tai271828** [comment]
Is the disk space still a blocker?  : )

File: backend/api/models/footprint_form.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+from django.contrib.postgres.fields import JSONField
+
+class FootprintForm():
+
+    id = models.AutoField(primary_key=True)
+    type = models.IntegerField(choices=[(0, '糞便'), (1, '腳印'), (2, '爪痕'), (3, '食痕'), (4, '其他')], blank=True, null=True) # 痕跡類型
+    type_desc = models.CharField(max_length=255, blank=True, null=True) # 痕跡類型描述
+    status = models.IntegerField(choices=[(0, '新'), (1, '舊'), (2, '不清楚')]) # 新舊狀態
+    status_desc = models.CharField(max_length=255, black=True, null=True) # 新舊狀態描述
+    image_uploaded = models.BooleanField(default=0) # 是否上傳圖檔
```
Comment (EagleC0318) [inline backend/api/models/footprint_form.py:12]: Looks good for me! 
Btw, why we use 0 instead of False for the boolean field?

Comment (tai271828) [reply backend/api/models/footprint_form.py:12]: Ah, good catch. Yeah, why do we use 0? : )

**Comment by yanghaochang104** [comment]
#### 3/30 本週無進度
- 持續了解Django中，並沒有特別發PR

File: backend/api/models/footprint_form.py (hunk 1 above)
Comment (yanghaochang104) [reply backend/api/models/footprint_form.py:12]: my bad XD
gotta fix it

**Comment by yanghaochang104** [comment]
> Is the disk space still a blocker? : )

waiting for my new machine(~4/1)
shifting side projects from my working computer to my personal computer recently

**Comment by tai271828** [comment]
> > Is the disk space still a blocker? : )
> 
> waiting for my new machine(~4/1) shifting side projects from my working computer to my personal computer recently

Sounds a really good timing because we moved the repositories to the "Github organization" and changed the default branch to "main" : D You are more than welcome to update the README after migrating your development environment this time. Good luck then!

**Comment by yanghaochang104** [comment]
刪改後對新的OH-SHOWN repository重發

**Comment by diaoga** [comment]
@yanghaochang104 ~Is everything about this PR okay?
We are waiting for this PR XDD

**Pull request closed without merging**